app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :comments_count
+  after_destroy :comments_count
+
+  private
+
+  # A method that updates the comments count for a post.
+  def comments_count
+    post.update(comments_count: post.comments.count)
+  end
```

Comment:
- [ ] I was unable to create a comment because there is no` comments_count` column in the schema and migration files; perhaps you meant `comments_counter`. Kindly correct this for proper execution of your project. ℹ️ 

<img width="567" alt="Untitled" src="https://github.com/lily4178993/nell-blog/assets/103386183/6dc4b9db-fb19-44cd-8f66-23b5402376e7">